packages/babel-generator/src/generators/methods.ts

```diff
@@ -11,7 +11,7 @@ export function _params(
   this._parameters(node.params, node);
   this.token(")");
 
-  this.print(node.returnType, node, node.type === "ArrowFunctionExpression");
+  this.print(node.returnType, node);
```

Comment:
I'm not sure if this is going to be removed, although we no longer wrap comments here.